src/css/components/_markdown-content.scss

```diff
@@ -113,14 +113,22 @@
     padding: 1px 5px;
 
     a {
-      @apply no-underline text-grey border-grey-dark;
+      @apply text-blue-dark no-underline border-b border-blue-light;
     }
 
     a:hover {
       @apply text-grey-lighter border-grey;
     }
   }
 
+  pre {
+    code {
+      a {
+        @apply no-underline text-grey border-grey-dark;
```

Comment:
Keep the links in the big code blocks as they were before.

Reply:
Love it! Thanks for the explanation